pipeline/plugins/operators/custom_operators.py

```diff
@@ -255,31 +479,23 @@ class PrepareDownloadLocationOperator(BaseOperator):
     def __init__(
         self,
         ssh_conn_id,
-        old_image_path,
+        old_target_path,
         ensure_dirs,
         extra_bin_dir,
         **kwargs,
     ):
         super().__init__(**kwargs)
         self.ssh_conn_id = ssh_conn_id
-        self.old_image_path = old_image_path
+        self.old_target_path = old_target_path
         self.ensure_dirs = ensure_dirs
         self.extra_bin_dir = extra_bin_dir
 
-    def extract_image(self, ssh_client):
-        """
-        Extract contents of a disk image in given path.
-        """
-        ssh_client.exec_command(
-            f"{self.extra_bin_dir}/unsquashfs -d {self.file_download_dir} {self.old_image_path}"
-        )
-
-    def create_file_listing_from_image(self, ssh_client):
+    def create_file_listing_from_target(self, ssh_client):
         """
-        Extract file listing of a disk image in given path to file_download_dir.
+        Extract file listing of a target in given path to file_download_dir.
         """
         ssh_client.exec_command(
-            f'{self.extra_bin_dir}/unsquashfs -d "" -lc {self.old_image_path} > {self.file_download_dir}/existing_files.txt'
+            f"unzip -l {self.old_target_path} > {self.file_download_dir}/existing_files.txt"
```

Comment:
This is now not used for anything, and we plan to not need it (because we will only fetch changed files), so I guess this should just be dropped.